docs/zh/latest/getting-started.md

```diff
@@ -170,6 +180,16 @@ curl "http://127.0.0.1:9080/apisix/admin/upstreams/1" -H "X-API-KEY: edd1c9f0343
 
 我们使用 `roundrobin` 作为负载均衡机制，并将 `httpbin.org:80` 设置为我们的上游服务，其 ID 为 `1`。更多字段信息，请参阅 [Admin API](./admin-api.md)。
 
+<!--
+#
```

Comment:
Do we need translate this tip to Chinese? @yzeng25

Reply:
It should be good for now.